src/imitation/scripts/common/rl.py

```diff
@@ -142,7 +155,17 @@ def load_rl_algo_from_path(
     rl_cls: Type[base_class.BaseAlgorithm],
     rl_kwargs: Mapping[str, Any],
     _seed: int,
+    relabel_reward_fn: Optional[RewardFn] = None,
 ) -> base_class.BaseAlgorithm:
+    rl_kwargs = dict(rl_kwargs)
+    if relabel_reward_fn and issubclass(
+        rl_cls,
+        off_policy_algorithm.OffPolicyAlgorithm,
+    ):
+        assert not isinstance(venv.observation_space, gym.spaces.Dict)
```

Comment:
A bit of code duplication here. Consider factoring out this into a helper method e.g. `_maybe_add_relabel_buffer`?

Reply:
yep